scripts/parser.js

```diff
@@ -0,0 +1,49 @@
+const db = require('../app/database/dbConfig.js');
+var MBO = require('mindbody-sdk');
+const { insertJob, updateJobs } = require('../app/models/jobsModel.js');
+require('dotenv').config();
+ 
+var mbo = new MBO({
+    ApiKey: process.env.API_KEY, // from portal
+    SiteId: -99 //thisis the sandbox account
+});
+
+/**
+ * Iterates through provided class information from Mindbody API
+ * and updates or inserts jobs to the database
+ */
+async function processResponse(err,data){
+    if (err) {
+        console.error("Failed to retrieve information about classes", err);
+    } else {
+        classes = data.Classes
+        for (var i = 0; i < classes.length; i++){
+            var session = classes[i]
+            scheduledJobs = await db('jobs')
```

Comment:
the reason we have the models folder is to separate any database work from the rest of the code base. if you need to get something from the database, make a new model function in the jobsModel, and then call it here